src/maximum_flow.jl

```diff
@@ -178,3 +178,19 @@ function maximum_flow(
     end
     return maximum_flow(flow_graph, source, target, capacity_matrix, algorithm)
 end
+
+"""
+    is_zero(value)
```

Comment:
You could replace `iszero` with `isapprox` directly, which handles both integers and abstract floats correctly

Reply:
We already discussed of that point, unfortunately eps is not defined for integers

> The default isapprox is not well suited for our problem. If `isapprox(0.2+0.1, 0.3)` is returning `true`, `isapprox(0.2+0.1-0.3, 0.0)` is returning `false`. In fact, `x ≈ 0` is equivalent to `x == 0`.
> As we compare to 0, rtol is just useless, so we must define an atol.
> If we want the tolerance to rely on the type, we must separate Float and Integer case (as the default rtol do), because eps is not defined for Integer types.
> This is not an absurd way o do it as rtol is defined quite similarly:
> 
> ```
> # default tolerance arguments
> rtoldefault(::Type{T}) where {T<:AbstractFloat} = sqrt(eps(T))
> rtoldefault(::Type{<:Real}) = 0
> function rtoldefault(x::Union{T,Type{T}}, y::Union{S,Type{S}}, atol::Real) where {T<:Number,S<:Number}
>     rtol = max(rtoldefault(real(T)),rtoldefault(real(S)))
>     return atol > 0 ? zero(rtol) : rtol
> end
> ```
> 
> See [Base.isapprox](https://docs.julialang.org/en/v1/base/math/#Base.isapprox)
> and [floatfuncs](https://github.com/JuliaLang/julia/blob/master/base/floatfuncs.jl) lines 285-291 for the definition of rtol